packages/core/utils/src/common/define-config.ts

```diff
@@ -0,0 +1,127 @@
+import { ConfigModule } from "@medusajs/types"
+import { Modules } from "@medusajs/modules-sdk"
+
+const DEFAULT_SECRET = "supersecret"
+const DEFAULT_ADMIN_URL = "http://localhost:9000"
+const DEFAULT_STORE_CORS = "http://localhost:8000"
+const DEFAULT_DATABASE_URL = "postgres://localhost/medusa-starter-default"
+const DEFAULT_ADMIN_CORS = "http://localhost:7000,http://localhost:7001"
```

Comment:
Adding the default vite port:

```suggestion
const DEFAULT_ADMIN_CORS = "http://localhost:7000,http://localhost:7001,http://localhost:5173"
```